dspace-api/src/main/java/org/dspace/app/itemexport/ItemExportCLITool.java

```diff
@@ -51,196 +49,175 @@
  * @author David Little
  * @author Jay Paz
  */
-public class ItemExportCLITool {
+public class ItemExportCLITool extends DSpaceRunnable<ItemExportScriptConfiguration> {
```

Comment:
In order to allow for different behavior from the CLI and the Admin UI, I think you'll need to split this class into two... there should be an ItemExportCLI class, and an ItemExport class.  See the `org.dspace.,app.bulkedit` classes (`MetadataExportCLI` and `MetadataExport`) as an example